apollo-client/src/main/java/com/ctrip/framework/apollo/spring/annotation/ApolloAnnotationProcessor.java

```diff
@@ -1,89 +1,138 @@
 package com.ctrip.framework.apollo.spring.annotation;
 
-import java.lang.reflect.Field;
-import java.lang.reflect.Method;
-
+import com.ctrip.framework.apollo.AutoConfigChangeListener;
+import com.ctrip.framework.apollo.Config;
+import com.ctrip.framework.apollo.ConfigChangeListener;
+import com.ctrip.framework.apollo.ConfigService;
+import com.ctrip.framework.apollo.model.ConfigChangeEvent;
+import com.google.common.base.Preconditions;
 import org.springframework.beans.BeansException;
 import org.springframework.beans.factory.config.BeanPostProcessor;
 import org.springframework.core.Ordered;
 import org.springframework.core.PriorityOrdered;
 import org.springframework.core.annotation.AnnotationUtils;
 import org.springframework.util.ReflectionUtils;
 
-import com.ctrip.framework.apollo.Config;
-import com.ctrip.framework.apollo.ConfigChangeListener;
-import com.ctrip.framework.apollo.ConfigService;
-import com.ctrip.framework.apollo.model.ConfigChangeEvent;
-import com.google.common.base.Preconditions;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
 
 /**
  * Apollo Annotation Processor for Spring Application
  *
  * @author Jason Song(song_s@ctrip.com)
  */
 public class ApolloAnnotationProcessor implements BeanPostProcessor, PriorityOrdered {
-  @Override
-  public Object postProcessBeforeInitialization(Object bean, String beanName) throws BeansException {
-    Class clazz = bean.getClass();
-    processFields(bean, clazz.getDeclaredFields());
-    processMethods(bean, clazz.getDeclaredMethods());
-    return bean;
-  }
+    @Override
```

Comment:
Please use Apollo's [code style](https://github.com/ctripcorp/apollo/tree/master/apollo-buildtools/style) so that we won't have code style differences.

Reply:
Thanks for your comment, I will reformat my code.